doc/swagger.rst

```diff
@@ -889,6 +889,41 @@ You can specify a custom validator URL by setting ``config.SWAGGER_VALIDATOR_URL
     api = Api(app)
 
 
+You can enable [OAuth2 Implicit Flow](https://oauth.net/2/grant-types/implicit/) for retrieving an
+authorization token for testing api endpoints interactively within Swagger UI.
+The ``config.SWAGGER_UI_OAUTH_CLIENT_ID`` and ``authorizationUrl`` and ``scopes``
+will be specific to your OAuth2 IDP configuration.
+The realm string is added as a query parameter to authorizationUrl and tokenUrl.
+These values are all public knowledge. No *client secret* is specified here.
+.. Using PKCE instead of Implicit Flow depends on https://github.com/swagger-api/swagger-ui/issues/5348
```

Comment:
I don't think it's a good idea to link an unresolved issue in the user documentation. I understand the reasoning behind it, however as it is not currently a feature, including this link makes the documentation more confusing. Furthermore, I can see it easily being forgotten - documentation is always hard to keep up to date (I've forgotten items like this many times :joy: ). Would you mind removing this line, then we can get this PR merged in. Thank you again!

Reply:
I intended this to be a comment (dot dot space arbitrary text) in the documentation source, explicitly to help future maintainers deal with such. Unless I am mistaken as to how comments work, this is not rendered in the consumer-visible result.

Does that change your feedback?

Reply:
Oh yes sorry, I apologise! I was not reading carefully enough :thinking:  I'm still concerned that the comment will get forgotten so I will link to this PR on that issue such that it's visible for when that issue is resolved. Thank you again !